### ISSUE TITLE
Collaborating on our similar projects

### ISSUE THREAD

**Issue by chartgerink** [rainsworth/ROSA#16]
@rainsworth it was good talking to you at OpenCon and am excited to collaborate on our projects! 🥂  

I am working from the [`libscie/now-boarding`](https://github.com/libscie/now-boarding) repository and would be happy to invite you as a maintainer there if you'd like. I think the main aspect of the collaboration is worthwhile on content and reviewing it via Pull Requests (where we can do line-by-line comments and iterate the content).

Our talk has motivated me to start drafting some content today (finally recovered from the conference), which you can view [here](https://github.com/libscie/now-boarding/blob/preprints/content/preprints.md). It's just a beginning, but I'd love for us to collaborate on creating the content. I can take on the task of the technical aspects such as the page around it. 

Let's discuss further here? 👍

**Comment by rainsworth** [comment]
@chartgerink  I am so excited about this! Yes, feel free to add me as a maintainer, I am happy to contribute/collaborate on content.

I especially like the format we talked about for presenting topics/content:
- introduce topic 
- provide info
- how-to
- resources
- exercises

as demonstrated on your preprint example above. 

We also talked about translations - @gedankenstuecke suggested getting in touch with [egle](https://twitter.com/seplute), who did community translation for https://stallcatchers.com/main in google docs.

**Comment by chartgerink** [comment]
@rainsworth 💯 Before talking to you I was getting a bit demotivated about this project but now am back on track. I just invited you onto the repo. Hm, I am wondering how we can best coordinate on topic creation and review. A spreadsheet maybe? Let me know what you think :-)

I just added the structure more explicitly to the CONTRIBUTING.md because I already forgot we talked about it like that 😮 

Translations is a good idea for when the content is stabilizing, so let's park that for now and come back to it in a few months? @gedankenstuecke I was thinking about something like [weblate](https://weblate.org/en/) at some point, you/egle have any experience with that?
